src/components/EventStats/EventStats.tsx

```diff
@@ -3,8 +3,9 @@ import { Modal } from 'react-bootstrap';
 import { FeedbackStats } from './Statistics/Feedback';
 import { ReviewStats } from './Statistics/Review';
 import { AverageRating } from './Statistics/AverageRating';
-import Stack from '@mui/material/Stack';
+// import Stack from '@mui/material/Stack';
 import styles from './Loader.module.css';
+import styling from './EventStats.module.css';
```

Comment:
```suggestion
import eventStatsStyles from './EventStats.module.css';
```
Always use this type of naming while importing css.